assets/i18n/qqq.json

```diff
@@ -42,6 +42,7 @@
 	"I18N_ACTION_BROWSE_LESSONS": "Text displayed in a button on the Home page. - When the user clicks the button they are taken to the library page. The text in the button needs to be less than 29 characters long.",
 	"I18N_ACTION_CREATE_EXPLORATION": "Text displayed in a button on the Home and Teach pages. - When the user clicks the button they are taken to their dashboard, and a dialog is opened where the user can create a new exploration.",
 	"I18N_ACTION_CREATE_LESSON": "Text displayed in a button on the Home page. - When the user clicks the button they are taken to their dashboard, and a dialog is opened where the user can create a new exploration. The text in the button needs to be less than 29 characters long.",
+	"I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE": "This card is quite long, and students might lose interest. Consider shortening it, or breaking it into two cards.",
```

Comment:
Over here you have to explain where, when and what it is used for. (See above lines for reference.)

Reply:
Oh my bad, Fixed this.